### PR TITLE
Added Homework 1

### PR DESCRIPTION
**Pull request by Hrist0ff**

### CODE REVIEW

File: homework/12_B/02_Aleksandar/homework1/circle.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  def check(self,other):    
+    dist = sqrt((self.x-other.x)**2 + (self.y-other.y)**2)
+    print (self.x)
+    print (other.x)
+    print (self.y)
+    print (other.y)
+
+    if self.radius <= 0 or other.radius <= 0:
+        return Intersection.INVALID.name # INVALID
```
Comment (svetlin-mladenov) [inline homework/12_B/02_Aleksandar/homework1/circle.py:26]: You can just return `Intersection.INVALID`

File: homework/12_B/02_Aleksandar/homework1/circle.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    if self.x == other.x and self.y == other.y and self.radius == other.radius:
+        return Intersection.SAME.name # SAME
+    elif (dist <= (self.radius - other.radius) or dist <= (other.radius - self.radius)):
+        return Intersection.INSIDE.name # INSIDE
+    elif dist == (self.radius + other.radius):
+        return Intersection.TOUCHING.name # TOUCHING
+    elif dist < (self.radius + other.radius):
+        return Intersection.INTERSECT.name # INTERSECT
```
Comment (svetlin-mladenov) [inline homework/12_B/02_Aleksandar/homework1/circle.py:34]: No need for the comments. The code make it clear.

File: homework/12_B/02_Aleksandar/homework1/circle.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+class Circle:
+  def __init__(self, x, y,radius):
+    self.x = x 
+    self.y = y
+    self.radius = radius
+
+  def check(self,other):    
+    dist = sqrt((self.x-other.x)**2 + (self.y-other.y)**2)
+    print (self.x)
```
Comment (svetlin-mladenov) [inline homework/12_B/02_Aleksandar/homework1/circle.py:20]: These are obviously debug statements. Please remove them.

File: homework/12_B/02_Aleksandar/homework1/test_circle.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+# https://www.desmos.com/calculator/6sntulfbz2
+
+def test_intersect():
+    assert C1.check(C3) == 'INTERSECT'
+
+def test_inside():
+    assert C1.check(C4) == 'INSIDE'
+
+def test_touching():
+    assert C1.check(C5) == 'TOUCHING'
```
Comment (svetlin-mladenov) [inline homework/12_B/02_Aleksandar/homework1/test_circle.py:19]: Can you add more test cases. For example when two circles are touching on the inside.

**Review by svetlin-mladenov — CHANGES_REQUESTED** [review]

**Review by Hrist0ff — COMMENTED** [review]
changes

**Pull request merged**